### PR TITLE
Add RaaS Roles

### PR DESCRIPTION
**Pull request by pixelwake**
IN PROGRESS

Adding the messaging for the new section to house the RaaS roles & links to position descriptions

### CODE REVIEW

**Comment by fureigh** [comment]
@rooneywp This one isn't yet ready for review, correct?

**Comment by fureigh** [comment]
@rooneywp Given our conversation in Slack, I'll assume you'd like me to review this.

Two initial things:
- The federal government benefits link referenced in all of these posts (https://www.usajobs.gov/ResourceCenter/Index/Interactive/Benefits#icc) yields a 404 error (see below). Can you update the link and check that all remaining links work as intended?

screen shot 2016-08-22 at 9 35 47 pm
- Several of the pull requests reviewed and merged were based off of other branches created for pull requests, not off of the main `18f-pages` branch, so things are a bit muddled at the moment. Let's sort those out, merge them into this branch, push it to staging, and then we'll be able to confirm we're good to go.

File: pages/open-positions.md [hunk 1]
```diff
@@ -9,6 +9,24 @@ We're briefly pausing accepting new applications while we evolve how 18F plans f
...
+## Opportunities with 18F partners
+We’re currently supporting other agencies’ recruiting efforts to build digital service and innovation teams. Below you’ll find a list of the current positions along with their associated agencies. For all of these opportunities, please note:
+- Priority deadline consideration for these roles is Friday, September 16, 2016.
+- You must be in the D.C. area or willing to relocate to D.C. Unfortunately, we can’t provide relocation assistance.
+
+If you’re interested in these opportunities, please email join18f@gsa.gov and reference the position you are interested in. You may apply to more than one role.
+
+**Environmental Protection Agency (EPA) Roles**
+- [Cloud Architect/Infrastructure Engineer](https://pages.18f.gov/joining-18f/raas/epa/cloud-architect/) 
+- [Technincal Architect, TAPD](https://pages.18f.gov/joining-18f/raas/epa/tapd-technical-architect/)
```
Comment (fureigh) [inline pages/open-positions.md]: @rooneywp When you have a sec: `Technincal` --> `Technical`

`18f-pages` sites use a different approach for linking; looking up an example elsewhere in this repo.

**Comment by pixelwake** [comment]
@fureigh Yeah, I thought something was weird. I must have done something wrong when I was creating the new files.

I'll update the benefits links.

File: pages/open-positions.md (hunk 1 above)
Comment (pixelwake) [reply pages/open-positions.md]: @fureigh Ok, happy to update when you get an example.

Comment (fureigh) [inline pages/open-positions.md]: ex: `[Application Security Engineer]({{ site.baseurl }}/infosec/application-security-engineer/)`

Source: line 12, https://github.com/18F/joining-18f/commit/b47a81c1f0e2ac3c5bec32f45c4e7e5d35059eba#diff-9d1fc2dd1711b601d6b47f341f8ddf16

Comment (fureigh) [inline pages/open-positions.md]: I'll get 'em this time.

**Comment by pixelwake** [comment]
@fureigh Don't worry about rushing to get this done tonight. We can get it up in the AM.

**Comment by fureigh** [comment]
@rooneywp Patience appreciated, but I'm in workshop mode for the rest of this week, so best to wrap it up tonight if possible. Aiming to get the branches sorted before this plane lands, after which you can make further edits at your convenience.

**Comment by fureigh** [comment]
Merged branch `rooneywp-patch-6`.

**Comment by fureigh** [comment]
Okay, branches appear to be sorted. I pushed to staging but 2FA is locking me out while I'm on a plane and don't have the ability to access texts. Problems of living in the future!

@rooneywp, can you review on [staging](https://pages-staging.18f.gov/joining-18f/)? In particular, please verify that the links all work and are formatted correctly.

**Comment by fureigh** [comment]
No pressure to do that tonight, of course; it's up to you. Everything is now in this PR and on staging, including the commits that were already merged to `18f-pages`.

**Pull request merged**